packages/iov-bns/src/types.ts

```diff
@@ -176,3 +194,153 @@ export const ensure = <T>(maybe: T | null | undefined, msg?: string): T => {
   }
   return maybe;
 };
+
+// transactions
+
+/**
+ * BNS transaction kinds.
+ *
+ * Raw values are used for type detection only and can change at any time.
+ */
+export enum TransactionKind {
```

Comment:
Huh? Why no TransactionKind in bcp, but there are here?

I could see getting rid of it everywhere, or using it everywhere, but why is it different?

Reply:
In this case I do not really use an enum but abuse `enum` when I need a namespace for string constants. I tried to convert to an object with string properties but those I can only use as values, not as types. Maybe there is a better way to implement it. What I really want is "mini namespace of strings"